models/Algo.py

```diff
@@ -0,0 +1,23 @@
+from Order import Order
+from ..utils.Alpaca_Data import AlpacaData
```

Comment:
I'm getting the problems here/

At the top directory, running `python cli.py` will cause import issues. I hate python imports. I still don't get them after three years.

Reply:
I got it to work. since every folder has a __init__.py, they are treated as modules and thus everything can be imported based on the name of the folder (aka module). so :
from models.Order import Order
from utils.Alpaca_Data import AlpacaData